homeassistant/components/depict/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""
+Support for controlling Depict digital art frames.
```

Comment:
We've moved away from multi line docstrings here. This should now read:

`"""Support for controlling Depict digital art frames."""